packages/dcos-integration-test/extra/test_endpoints.py

```diff
@@ -191,6 +209,28 @@ def test_if_overlay_master_agent_is_up(cluster):
     except KeyError as ex:
         raise AssertionError("Could not find key 'subnet':" + str(ex)) from ex
 
+    # Pop mesos and docker bridge if they have been configured on the
```

Comment:
It's alright. I realize it was decided to use the pattern of `pop-out` values and do a `dict` compare at the end.